app/presenters/nsm/check_answers/cost_summary_card.rb

```diff
@@ -138,17 +140,28 @@ def vat_rate
         @vat_rate ||= claim.firm_office.vat_registered == 'yes' ? Pricing.for(claim).vat : 0.0
       end
 
-      def format(value, formatted: true)
+      def format(value, formatted: true, accessibility_key: '')
         return value unless formatted
         return nil if value.nil?
 
-        { text: NumberTo.pounds(value), numeric: true }
+        if accessibility_key.present?
+          { text: accessible_text(accessibility_key, NumberTo.pounds(value)), numeric: true }
+        else
+          { text: NumberTo.pounds(value), numeric: true }
+        end
+      end
+
+      def accessible_text(key, value)
+        scope = show_adjustments ? 'with_adjustments' : 'base'
+
+        text = I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.accessibility.#{key}")
+        sanitize("#{govuk_visually_hidden(text)} #{value}")
       end
 
       def t(key, numeric: true, width: nil)
         scope = show_adjustments ? 'with_adjustments' : 'base'
         {
-          text: I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.#{key}"),
+          text: I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.#{key}").html_safe,
```

Comment:
we shouldn't need to add html_safe here... if it needs to contain html you should postfix the key with `_html`

Reply:
is is postfixed with _html but renders escaped html without this - i think because this is not the view but a precursor to the rendered view